Rename Analytics rules to Smart rules (New)

nnat-nabu opened the pull request:
This PR introduces updates across multiple components, including Streaming Analytics , Cockpit, and Device Management. It also adds the corresponding changelog entries and restructures Cockpit entries so they appear as top-level items.
Related to: https://cumulocity.atlassian.net/browse/PAB-4907

This PR is to be merged once the rename from analyticsrules in the shell applications (cockpit etc) has reached eu-latest.

nnat-nabu committed:
rename

nnat-nabu committed:
file names updated

nnat-nabu committed:
smart rules 2.0

nnat-nabu committed:
2.0 stylings

nnat-nabu committed:
Revert "2.0 stylings"

This reverts commit 5e775b552c671dd3a0212c5e48b87369206f3def.

nnat-nabu committed:
changelog

nnat-nabu committed:
Changelog

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+### Creating your first smart rules 2.0 {#creating-your-first-smart-rules-2.0}
 
-This topic provides the basic workflow for creating your first analytics rule using the plugin from applications like Device Management. You will create a simple templated analytics model that creates an alarm when the temperature breaches a given threshold value. The steps below require that at least one device is already registered in {{< product-c8y-iot >}}. Preferably, this device already sen ...
+This topic provides the basic workflow for creating your first Smart rules 2.0 using the plugin from applications like Device Management. You will create a simple templated analytics model that creates an alarm when the temperature breaches a given threshold value. The steps below require that at least one device is already registered in {{< product-c8y-iot >}}. Preferably, this device already se ...
```
See the story A/C - we only need to use "2.0"/"NEW" in the top-level heading - in most of the text (e.g. here) we can just call them "smart rules"

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -60,27 +60,27 @@ When completed, your model will look similar to this:
+**Congratulations!** You have successfully created and deployed your first smart rule using the Smart rules 2.0 plugin.
 
-### Creating Analytics rules for existing samples {#create-analytics-rule-for-existing-samples}
+### Creating Smart rules 2.0 for existing samples {#create-smart-rule-for-existing-samples}
```
```suggestion
### Creating Smart rules from existing Analytics Builder samples {#create-smart-rule-for-existing-samples}
```

I think making it clear the samples are in AB is useful

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -19,10 +19,10 @@ This process requires switching between applications and can be time-consuming w
 
-#### With Analytics rules plugin {#with-analytics-rules-plugin}
-The Analytics rules plugin streamlines this process by enabling analytics builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to different application.
+#### With Smart rules 2.0 plugin {#with-smart-rules-plugin}
```
Smaller headings like this can just be "Smart rules" without the suffix

ben-c8y committed:
Rewrite changelog for new naming

Changes from Analytics rules to Smart rules (NEW), includes a brief comparison to the existing smart rules, and also mentions the DTM functionality

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
 ---
 weight: 10
-title: Getting started with the Analytics rules plugin
+title: Getting started with the Smart rules 2.0 plugin
```
```suggestion
title: Getting started with the Smart rules (NEW) plugin
```

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
 
-### What is the Analytics rules plugin {#what-is-the-analytics-rules-plugin}
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
+### What is the Smart rules 2.0 plugin {#what-is-the-smart-rules-2.0-plugin}
```
```suggestion
### What is the Smart rules plugin {#what-is-the-smart-rules-plugin}
```

We can just use "Smart rules" here, the top-level title is enough to show we're talking about the "new" one

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
-### What is the Analytics rules plugin {#what-is-the-analytics-rules-plugin}
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
+### What is the Smart rules 2.0 plugin {#what-is-the-smart-rules-2.0-plugin}
+The Smart rules 2.0 plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
```
```suggestion
The Smart rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
```

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
 ### Prerequisites {#prerequisites}
 
-Before using the Analytics rules plugin, ensure that the following requirements are met:
+Before using the Smart rules 2.0 plugin, ensure that the following requirements are met:
```
```suggestion
Before using the Smart rules (NEW) plugin, ensure that the following requirements are met:
```
It's worth saying "new" here, since it's the first mention of it, but after this we don't need to say new

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+Before using the Smart rules 2.0 plugin, ensure that the following requirements are met:
 - The tenant is subscribed to the Streaming Analytics microservice that supports the Analytics Builder capability.
-- The Analytics rules plugin is installed in the desired application, and the feature flag `streaming-analytics.analytics-rules-plugin` is enabled.
+- The Smart rules 2.0 plugin is installed in the desired application, and the feature flag `streaming-analytics.smart-rules-2.0-plugin` is enabled.
```
```suggestion
- The new Smart rules plugin is installed in the desired application, and the feature flag `streaming-analytics.smart-rules-plugin` is enabled.
```

NB: need to make sure we've renamed the feature flag to match this documentation. Not sure if we need a changelog for that

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/index.html:
```diff
@@ -0,0 +1,4 @@
+---
+title: Smart rules 2.0 plugin
```
```suggestion
title: Smart rules (NEW) plugin
```

ben-c8y committed:
typo

nnat-nabu committed:
remamed to (NEW)

nnat-nabu committed:
refactor to (NEW)

nnat-nabu committed:
refactor

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+Before using the Smart rules (NEW) plugin, ensure that the following requirements are met:
 - The tenant is subscribed to the Streaming Analytics microservice that supports the Analytics Builder capability.
-- The Analytics rules plugin is installed in the desired application, and the feature flag `streaming-analytics.analytics-rules-plugin` is enabled.
+- The new Smart rules plugin is installed in the desired application, and the feature flag `streaming-analytics.smart-rules-plugin` is enabled.
```
We won't change the feature flag id, but anyway we shuold mention the display name not the id here

nnat-nabu replied:
I don't get the display name. It's a boolean flag. Could you please suggest me the content?

ben-c8y replied:
Thinking like customer, what woudl they actually do to enable it?

Answer; go to "Preview features" and enable the "Smart rules (NEW) plugin" e.g. 

<img width="1855" height="961" alt="image" src="https://github.com/user-attachments/assets/c544a983-9a45-4dea-9ea7-3bc98ca64337" />

So they don't need to know the id of the feature flag

nnat-nabu replied:
updated

nnat-nabu committed:
changelog refactor

nnat-nabu committed:
refactor

nnat-nabu committed:
fix changelog mismatch

ben-c8y commented on content/change-logs/device-management/ui-c8y-new-analytics-rules-plugin-as-public-preview.md:
```diff
@@ -1,6 +1,6 @@
 date: '2025-08-28'
-title: Analytics rules plugin now available in Public Preview
+title: Smart rules (NEW) plugin now available in Public Preview
 product_area: Device management & connectivity
```
Please use the title and text from this change log in the other changelogs as well (e.g. the mention of Digital Twin manager) etc. If we end up needing using 2 PRs we can move this edit into the other PR

ben-c8y commented on content/change-logs/device-management/ui-c8y-new-analytics-rules-plugin-as-public-preview.md:
```diff
@@ -15,10 +15,12 @@ ticket: MTM-64266
-The Analytics rules plugin is now available in Public Preview. It extends the capabilities of the Analytics Builder in Streaming Analytics, allowing users to define analytics rules and create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit and Device Management.
-
-For more details about the Analytics rules plugin, refer to [Analytics rules plugin](/streaming-analytics/analytics-rules-plugin#what-is-analytics-rules-plugin).
+For more details about the Smart rules plugin, refer to [Analytics rules plugin](/streaming-analytics/analytics-rules-plugin#what-is-analytics-rules-plugin).
```
not analytics-rules-plugin

skom-c8y replied:
also, please make sure the link 'id' is correct, in getting-started, you have the section name labelled as `### What is the Smart rules plugin {#what-is-the-smart-rules-plugin}` which means the link id should be `#what-is-the-smart-rules-plugin` but here you are missing 'the'

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
-### What is the Analytics rules plugin {#what-is-the-analytics-rules-plugin}
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
+### What is the Smart rules plugin {#what-is-the-smart-rules-plugin}
+The Smart rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
```
and Digital Twin Manager

nnat-nabu committed:
added DTM

nnat-nabu committed:
typo

nnat-nabu committed:
resolve conflicts

nnat-nabu committed:
changelog updated for streaming analytics

nnat-nabu committed:
Changelogs updated

nnat-nabu committed:
Update anchor

Co-authored-by: Ben Spiller <84027153+ben-c8y@users.noreply.github.com>

nnat-nabu committed:
refactor links

nnat-nabu committed:
fix links

nnat-nabu committed:
resolve conflicts

nnat-nabu committed:
updated screenshots link

nnat-nabu committed:
update screenshots link

nnat-nabu committed:
image dir name update

nnat-nabu committed:
resolve conflicts with other PR

nnat-nabu committed:
update help content

ben-c8y commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+Before using the Smart rules (NEW) plugin, ensure that the following requirements are met:
 - The tenant is subscribed to the Streaming Analytics microservice that supports the Analytics Builder capability.
-- The Analytics rules plugin is installed in the desired application, and the feature flag `streaming-analytics.analytics-rules-plugin` is enabled.
+- The new Smart rules plugin is installed in the desired application, and the **Smart rules (NEW) plugin** is enabled in **Preview feature**.
```
```suggestion
- The Smart rules plugin is installed in the desired application, and the **Smart rules (NEW) plugin** is enabled in **Preview feature**.
```

ben-c8y committed:
Update content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md

ben-c8y commented on content/cockpit/smart-rules-new.md:
```diff
@@ -0,0 +1,24 @@
+---
+weight: 80
```
could we rename this file to smart-rules-plugin.md instead of smart-rules-new.md? Then we don't have to change it later when we remove the "new" (but since "plugin" only applies to the new one, it's clear to us that it doesn't refer to the old one)

nnat-nabu replied:
A file smart-rules-plugin.md already exists by that name. It picks up the content from that file. We need a distinct name

ben-c8y replied:
The only other file I see with that name is part of streaming-analytics not cockpit. Are you sure we can't create a file named smart-rules-plugin.md in cockpit?

nnat-nabu replied:
renamed to cockpit-smart-rules-plugin.md as discussed

nnat-nabu committed:
refactor weight

nnat-nabu committed:
Merge branch 'develop' into dev/PAB-4907-rename-analytics-rules-to-smart-rules-2.0

nnat-nabu committed:
Merge branch 'develop' into dev/PAB-4907-rename-analytics-rules-to-smart-rules-2.0

nnat-nabu committed:
rename file

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20250603-New-Analytics-rules-plugin-is-available-as-public-preview.md:
```diff
@@ -16,9 +16,9 @@ version: 26.152.0
 {{< /c8y-admon-preview >}}
 
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage the Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit. You can activate the feature using the **Manage preview features** option in the user menu.
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
```
```suggestion
While traditional smart rules provide a fixed set of predefined templates for common scenarios, the new generation of smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit, Device Management, and Digital Twin Manager.
```

BeateRixen commented on content/change-logs/application-enablement/ui-c8y-new-analytics-rules-plugin-in-public-review.md:
```diff
@@ -16,9 +16,9 @@ version: 1022.18.0
 {{< /c8y-admon-preview >}}
 
-The Analytics rules plugin is now available in Public Preview. It extends the capabilities of the Analytics Builder in Streaming Analytics, allowing users to define analytics rules and create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit and Device Management.
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
```
```suggestion
While traditional smart rules provide a fixed set of predefined templates for common scenarios, the new generation of smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit, Device Management, and Digital Twin Manager.
```

BeateRixen commented on content/change-logs/device-management/ui-c8y-new-analytics-rules-plugin-as-public-preview.md:
```diff
@@ -15,10 +15,14 @@ ticket: MTM-64266
 
+The Smart rules (NEW) plugin is now available in Public Preview. 
+
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
```
```suggestion
While traditional smart rules provide a fixed set of predefined templates for common scenarios, the new generation of smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit, Device Management, and Digital Twin Manager.
```

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20250603-New-Analytics-rules-plugin-is-available-as-public-preview.md:
```diff
@@ -1,6 +1,6 @@
 ---
 date: '2025-07-17'
```
This has been published some time ago. Isn´t a simple renaming confusing? Wouldn´t it be better to add an extra change log for the renaming?

nnat-nabu replied:
Since we recently published the change log and the feature is still in "feature preview", we decided to update the existing change log instead of creating a new one.
@ben-c8y, could you please share your thoughts?

ben-c8y replied:
yes I agree it's a bit of a tricky one, but we've handled it this way because it's only introduced 2 months ago (not very long yet) and still in preview. We definnitely wanted the existing change log that announces it to be updated to avoid confusion (if someone hasn't spotted it already, then reads changelog saying we've introduced "Analytics rules" then cant' find that anywhere it'd be weird). 

I'm open to adding an additional change log for the rename if you think it's really needed, but since this probably hasn' been used by so many people yet our idea was to keep teh rename low-key. Is that ok or do you think we really need an extra changelog?

BeateRixen replied:
No, this is sufficient then. And probably the less confusing variant.

BeateRixen commented on content/change-logs/device-management/ui-c8y-new-analytics-rules-plugin-as-public-preview.md:
```diff
@@ -15,10 +15,14 @@ ticket: MTM-64266
 {{< /c8y-admon-preview >}}
 
-The Analytics rules plugin is now available in Public Preview. It extends the capabilities of the Analytics Builder in Streaming Analytics, allowing users to define analytics rules and create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit and Device Management.
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
```
Duplicate content

BeateRixen commented on content/cockpit/cockpit-smart-rules-plugin.md:
```diff
@@ -0,0 +1,24 @@
+This feature is in Public Preview and may be subject to change in the future.
+{{< /c8y-admon-preview >}}
+
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
```
```suggestion
While traditional smart rules provide a fixed set of predefined templates for common scenarios, the new generation of smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and device contexts within applications such as Cockpit, Device Management, and Digital Twin Manager.
```

BeateRixen commented on content/device-management-application/viewing-device-details.md:
```diff
@@ -38,19 +38,20 @@ Details on these additional menu items are provided where required.
 The **Alarms** tab provides information on the alarms of a device. See [Working with alarms](/device-management-application/monitoring-and-controlling-devices/#working-with-alarms) for detailed information on alarms.
 
-### Analytics rules {#analytics-rules}
+### Smart rules (NEW) {#smart-rules}
```
```suggestion
### Smart rules (NEW) {#smart-rules-new}
```

ben-c8y replied:
We don't want to do this change as we're really keen to avoid baking "new" into the doc more than we have to, since we will be removing the "new" once we get it to GA and deprecate and remove the traditional smart rules. We use "(NEW)" for top-level headings to make clear it's different from old smart rules, but I don't want it in places like identifiers or detailed prose where it's not truly essential

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
-### What is the Analytics rules plugin {#what-is-the-analytics-rules-plugin}
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
+### What is the Smart rules plugin {#what-is-the-smart-rules-plugin}
+The Smart rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management, Cockpit and Digital Twin Manager.
```
```suggestion
The Smart rules (NEW) plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management, Cockpit, and Digital Twin Manager.
```

ben-c8y replied:
We don't want to do this kind of change as we're already in the contetx of new smart rules - we have a PM decision not to keep repeating NEW all over the documentation text in places where the context is already established

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
 
-### What is the Analytics rules plugin {#what-is-the-analytics-rules-plugin}
-The Analytics rules plugin extends the Analytics Builder capabilities within Streaming Analytics by enabling users to create and manage Analytics Builder model instances directly from device and group contexts within applications such as Device Management and Cockpit.
+### What is the Smart rules plugin {#what-is-the-smart-rules-plugin}
```
```suggestion
### What is the Smart rules (NEW) plugin {#what-is-the-smart-rules-plugin}
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+Before using the Smart rules (NEW) plugin, ensure that the following requirements are met:
 - The tenant is subscribed to the Streaming Analytics microservice that supports the Analytics Builder capability.
-- The Analytics rules plugin is installed in the desired application, and the feature flag `streaming-analytics.analytics-rules-plugin` is enabled.
+- The Smart rules plugin is installed in the desired application, and the **Smart rules (NEW) plugin** is enabled in **Preview feature**.
```
```suggestion
- The Smart rules (NEW) plugin is installed in the desired application, and the **Smart rules (NEW) plugin** is enabled in **Preview feature**.
```

ben-c8y replied:
We're not using the word "NEW" for the plugin since hte old smart rules doesn't have a plugin, and the plugin name can't be changed later without affecting customers, so this change shouldn't be made

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -1,24 +1,24 @@
+### Creating your first smart rule {#creating-your-first-smart-rule}
 
-This topic provides the basic workflow for creating your first analytics rule using the plugin from applications like Device Management. You will create a simple templated analytics model that creates an alarm when the temperature breaches a given threshold value. The steps below require that at least one device is already registered in {{< product-c8y-iot >}}. Preferably, this device already sen ...
+This topic provides the basic workflow for creating your first Smart rules using the plugin from applications like Device Management. You will create a simple templated analytics model that creates an alarm when the temperature breaches a given threshold value. The steps below require that at least one device is already registered in {{< product-c8y-iot >}}. Preferably, this device already sends  ...
```
```suggestion
This topic provides the basic workflow for creating your first smart rules using the plugin from applications like Device Management. You will create a simple templated analytics model that creates an alarm when the temperature breaches a given threshold value. The steps below require that at least one device is already registered in {{< product-c8y-iot >}}. Preferably, this device already sends measurement values to {{< product-c8y-iot >}}.
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -50,42 +50,42 @@ This section walks you through creating a simple temperature monitoring model. F
+![Create model for alarms](/images/streaming-analytics/smart-rules-plugin/create-alarm-model.png)
 
-#### Step 2: Create model instance using Analytics rules plugin {#create-an-instance-of-model-using-analytics-rules-plugin}
+#### Step 2: Create model instance using Smart rules plugin {#create-an-instance-of-model-using-smart-rules-plugin}
```
```suggestion
#### Step 2: Create a model instance using the Smart rules (NEW) plugin {#create-an-instance-of-model-using-smart-rules-plugin}
```

ben-c8y replied:
We can change teh capitalization of Smart->smart but don't want to add NEW in this heading

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -50,42 +50,42 @@ This section walks you through creating a simple temperature monitoring model. F
-4. Click **Add rule** to open the **Add analytics rule** dialog. Since we are currently in the **Device** context, only models that have been configured with **From Context** template parameters restricted to **Device** will be visible here. Select your configured model and click **OK**.
-   ![Add rule for the selected model](/images/streaming-analytics/analytics-rules-plugin/add-rule-for-selected-model.png)
+   ![Smart rules tab](/images/streaming-analytics/smart-rules-plugin/empty-smart-rule.png)
+4. Click **Add rule** to open the **Add rule** dialog. Since we are currently in the **Device** context, only models that have been configured with **From Context** template parameters restricted to **Device** will be visible here. Select your configured model and click **OK**.
```
```suggestion
4. Click **Add rule** to open the **Add rule** dialog. Since you are currently in the **Device** context, only models that have been configured with **From Context** template parameters restricted to **Device** will be visible here. Select your configured model and click **OK**.
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -50,42 +50,42 @@ This section walks you through creating a simple temperature monitoring model. F
    - Click **Save** to save the rule, or toggle to **Active** to save and deploy immediately.
-6. Your rule now appears in the Analytics rules list, showing its status (Active/Inactive).
-   ![Analytics rules list](/images/streaming-analytics/analytics-rules-plugin/analytics-rules-list.png)
+6. Your rule now appears in the Smart rules list, showing its status (Active/Inactive).
```
```suggestion
6. Your rule now appears in the smart rules list, showing its status (Active/Inactive).
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -50,42 +50,42 @@ This section walks you through creating a simple temperature monitoring model. F
+**Congratulations!** You have successfully created and deployed your first smart rule using the Smart rules plugin.
 
-### Creating Analytics rules for existing samples {#create-analytics-rule-for-existing-samples}
+### Creating Smart rules from existing Analytics Builder samples {#create-smart-rule-for-existing-samples}
```
```suggestion
### Creating smart rules from existing Analytics Builder samples {#create-smart-rule-for-existing-samples}
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -94,17 +94,17 @@ You can also create analytics rules using pre-built sample models from Analytics
    - Ensure that the Analytics Builder models have template parameters configured with the "From Context" value.
    - Verify your current context matches the model's "Source or Destination" type restrictions. For example, when in a device context where the model's **Restrict to** dropdown is configured with "Groups" only, that model will not be available. To appear in device contexts, "Device" must be selected in the **Restrict to** dropdown.
-2. **Previously created rules don't appear in the Analytics rules list**
+2. **Previously created rules don't appear in the Smart rules list**
```
```suggestion
2. **Previously created rules don't appear in the smart rules list**
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/getting-started.md:
```diff
@@ -94,17 +94,17 @@ You can also create analytics rules using pre-built sample models from Analytics
    - Check if Analytics instance names are set in the Analytics Builder instance editor.
    - Confirm you're viewing the correct device or group context.
-3. **Error in Analytics rule deployment**
+3. **Error in Smart rule deployment**
```
```suggestion
3. **Error in smart rule deployment**
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -1,10 +1,10 @@
 ---
 
-### Analytics Builder workflow versus Analytics rules plugin {#analytics-builder-workflow-versus-analytics-rules-plugin}
+### Analytics Builder workflow versus Smart rules plugin {#analytics-builder-workflow-versus-smart-rules-plugin}
```
```suggestion
### Analytics Builder workflow versus Smart rules (NEW) plugin {#analytics-builder-workflow-versus-smart-rules-plugin}
```

ben-c8y replied:
We don't want to do this change as we're already in the contetx of new smart rules

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -19,10 +19,10 @@ This process requires switching between applications and can be time-consuming w
 
-#### With Analytics rules plugin {#with-analytics-rules-plugin}
-The Analytics rules plugin streamlines this process by enabling analytics builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to different application.
+#### With Smart rules plugin {#with-smart-rules-plugin}
```
```suggestion
#### With Smart rules (NEW) plugin {#with-smart-rules-plugin}
```

ben-c8y replied:
We don't want to do this change as we're already in the contetx of new smart rules

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -19,10 +19,10 @@ This process requires switching between applications and can be time-consuming w
-#### With Analytics rules plugin {#with-analytics-rules-plugin}
-The Analytics rules plugin streamlines this process by enabling analytics builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to different application.
+#### With Smart rules plugin {#with-smart-rules-plugin}
+The Smart rules plugin streamlines this process by enabling analytics builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to different application.
```
```suggestion
The smart rules plugin streamlines this process by enabling Analytics Builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to a different application.
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -19,10 +19,10 @@ This process requires switching between applications and can be time-consuming w
+The Smart rules plugin streamlines this process by enabling analytics builder model instance creation and deployment directly from device and group contexts, eliminating the need for switching to different application.
 
-### Understanding the Analytics rules interface {#understanding-the-analytics-rules-interface}
+### Understanding the Smart rules interface {#understanding-the-smart-rules-interface}
```
```suggestion
### Understanding the smart rules interface {#understanding-the-smart-rules-interface}
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -38,11 +38,11 @@ A rule has the following fields and properties:
 ### Integration with Analytics Builder instance editor {#integration-with-analytics-builder-instance-editor}
 
-The Analytics rules plugin maintains seamless bidirectional integration with the Analytics Builder instance editor.
+The Smart rules plugin maintains seamless bidirectional integration with the Analytics Builder instance editor.
```
```suggestion
The Smart rules (NEW) plugin maintains seamless bidirectional integration with the Analytics Builder instance editor.
```

ben-c8y replied:
We don't want to do this change as we're already in the contetx of new smart rules

BeateRixen commented on content/streaming-analytics/smart-rules-plugin-bundle/understanding-the-plugin.md:
```diff
@@ -38,11 +38,11 @@ A rule has the following fields and properties:
+Rules created in the Analytics Builder instance editor appear in the Smart rules plugin only when the mode is set to **Production**. Non-production modes indicate the rule is under development or testing and not ready for deployment.
 
-Rules created through the Analytics rules plugin automatically appear in the corresponding model's instance editor. These rules can be edited, modified, or deleted from the instance editor.
+Rules created through the Smart rules plugin automatically appear in the corresponding model's instance editor. These rules can be edited, modified, or deleted from the instance editor.
```
```suggestion
Rules created through the smart rules plugin automatically appear in the corresponding model's instance editor. These rules can be edited, modified, or deleted from the instance editor.
```

BeateRixen commented on content/streaming-analytics/smart-rules-plugin.md:
```diff
@@ -0,0 +1,14 @@
+helpcontent:
+- label: smart-rules-plugin
+  title: Smart rules (NEW)
+  content: "The Smart rules plugin extends the capabilities of Analytics Builder within Streaming Analytics. It allows users to create, deploy, and manage Analytics Builder models directly from asset, device and group contexts in applications like Device Management, Cockpit and Digital Twin Manager."
```
```suggestion
  content: "The smart rules plugin extends the capabilities of Analytics Builder within Streaming Analytics. It allows users to create, deploy, and manage Analytics Builder models directly from asset, device, and group contexts in applications like Device Management, Cockpit, and Digital Twin Manager."
```

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20250603-New-Analytics-rules-plugin-is-available-as-public-preview.md:
```diff
@@ -16,9 +16,9 @@ version: 26.152.0
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
 
-For more details about the Analytics rules plugin, refer to [Analytics rules plugin](https://cumulocity.com/docs/streaming-analytics/analytics-rules-plugin#what-is-the-analytics-rules-plugin).
+For more details about the Smart rules plugin, refer to [Smart rules plugin](https://cumulocity.com/docs/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```
```suggestion
For more details about the Smart rules (NEW) plugin, refer to [Smart rules (NEW) plugin](https://cumulocity.com/docs/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```

BeateRixen commented on content/change-logs/application-enablement/ui-c8y-new-analytics-rules-plugin-in-public-review.md:
```diff
@@ -16,9 +16,9 @@ version: 1022.18.0
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
 
-For more details about the Analytics rules plugin, refer to [Analytics rules plugin](/streaming-analytics/analytics-rules-plugin#what-is-the-analytics-rules-plugin).
+For more details about the Smart rules plugin, refer to [Smart rules plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```
```suggestion
For more details about the Smart rules (NEW) plugin, refer to [Smart rules (NEW) plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```

BeateRixen commented on content/change-logs/device-management/ui-c8y-new-analytics-rules-plugin-as-public-preview.md:
```diff
@@ -15,10 +15,14 @@ ticket: MTM-64266
+While traditional Smart rules provide a fixed set of predefined templates for common scenarios, the new generation of Smart rules removes these limitations by enabling you to create fully customized rules tailored to your specific requirements using the capabilities of Analytics Builder in Streaming Analytics. This allows you to create and manage rule instances directly from asset, group, and dev ...
 
-For more details about the Analytics rules plugin, refer to [Analytics rules plugin](/streaming-analytics/analytics-rules-plugin#what-is-the-analytics-rules-plugin).
+For more details about the Smart rules plugin, refer to [Smart rules plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```
```suggestion
For more details about the Smart rules (NEW) plugin, refer to [Smart rules (NEW) plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```

BeateRixen commented on content/cockpit/managing-assets-bundle/working-with-assets.md:
```diff
@@ -51,7 +51,7 @@ Note that global inventory permissions override inventory role permissions.
 
 - **Smart rules** - Shows smart rules specified for the device, see also [Smart rules](/cockpit/smart-rules/).
-- **Analytics rules** - Shows analytics rules specified for the device, see also [Analytics rules plugin](/streaming-analytics/analytics-rules-plugin#what-is-the-analytics-rules-plugin).
+- **Smart rules (NEW)** - Shows smart rules (NEW) specified for the device, see also [Smart rules plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```
```suggestion
- **Smart rules (NEW)** - Shows smart rules (NEW) specified for the device, see also [Smart rules (NEW) plugin](/streaming-analytics/smart-rules-plugin#what-is-the-smart-rules-plugin).
```

nnat-nabu committed:
remove duplicate content

nnat-nabu committed:
update we to you

Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>

ben-c8y committed:
Apply suggestions from Beate's review

Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>

nnat-nabu committed:
update Smart to smart

nnat-nabu committed:
screenshots update

nnat-nabu committed:
cockpit-group screenshot

BeateRixen commented:
@ben-c8y I will approve as is. But we need to find a better solution than publishing the same change log entry three times in a row. I understand that the reason for this is the 3 different components, but that is not obvious at first sight when you read the change logs without any filtering turned on.
